user/enterprise/build-images.md

```diff
@@ -65,7 +65,7 @@ you can run:
 
 ### Worker machine configuration
 
-In order to build other docker images, the Worker needs to be setup to support Docker Builds. To get started, please add the following to `/etc/default/travis-worker` on all your Workers:
+To build docker images on Travis CI Enterprise, add the following to /etc/default/travis-worker on all your Workers:
```

Comment:
And I'd leave the path in code as you had it before `/etc/default/travis-worker`

Reply:
oops - this was definitely not intended 😅 Will fix that right away